client/src/qa/components/QuestionAndAnswerEntry.jsx

```diff
@@ -1,26 +1,59 @@
-import React, { useState } from 'react';
-import styled from 'styled-components';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+// import styled from 'styled-components';
 import QuestionEntry from './QuestionEntry';
 import AnswerEntry from './AnswerEntry';
 
 const QuestionAndAnswerEntry = ({ question, answers }) => {
-  const [slice, setSlice] = useState(2);
-  const initialAnswers = answers.slice(0, slice);
+  const [slicer, setSlice] = useState(2);
+  const initialAnswers = answers.slice(0, slicer);
   const [currentAnswers, setCurrentAnswers] = useState(initialAnswers);
   const numOfAnswersLeft = answers.length - currentAnswers.length;
+  const showMoreAnswers = () => numOfAnswersLeft > 0;
```

Comment:
I noticed what you did here. Pretty good idea!

Reply:
🥇